DataProcessing/datax-host/src/main/scala/datax/host/BlobBatchingHost.scala

```diff
@@ -102,5 +102,35 @@ object BlobBatchingHost {
 
     appLog.warn(s"Batch Mode Work Ended, processed metrics: $batchResult")
     AppInsightLogger.trackEvent(ProductConstant.ProductRoot + "/batch/end", null, batchResult)
+
+    //write tracker file
+    val (ok, trackerFolder, dateTime) = getTrackerConfigs(inputArguments)
+    if (ok) {
+      writeTracker(trackerFolder, dateTime)
+    }
+  }
+
+  def getTrackerConfigs(inputArguments: Array[String]) : (Boolean, String, Timestamp) = {
+    val namedArgs = ArgumentsParser.getNamedArgs(inputArguments)
+    val inputRoot = namedArgs.getOrElse("trackerFolder", "")
+
+    if (inputRoot == "") {
+      (false, "", null)
+    } else {
+      val timeStart = sys.env.getOrElse("process_start_datetime", "")
+      (true, inputRoot, Timestamp.from(Instant.parse(timeStart)))
```

Comment:
if trackerFolder is given in the input arguments, then it writes a tracker file under that folder